src/Components/Carousel/Carousel.module.css

```diff
@@ -0,0 +1,37 @@
+.containerCarousel {
```

Comment:
this styles should come from Styles folder and on the styles folder should be separated by components

Reply:
changes added